Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+                                    stepActivity.AddVariable(stepActivityVar);
+                                    stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because new flow control param was added
+                                }
+                                else
+                                {
+                                    //add as String param
+                                    stepActivityVar = new VariableString();
+                                    stepActivityVar.Name = param;
+                                    ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
+                                    stepActivity.AddVariable(stepActivityVar);
+                                }
+                            }
+                            else
+                            {
+                                //#param exist
+                                if (isflowControlParam == true)
+                                {
+                                    if (!(stepActivityVar is VariableSelectionList))
+                                    {
+                                        //flow control param must be Selection List so transform it
+                                        stepActivity.Variables.Remove(stepActivityVar);
+                                        stepActivityVar = new VariableSelectionList();
+                                        stepActivityVar.Name = param;
+                                        stepActivity.AddVariable(stepActivityVar);
+                                        stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because flow control param was added
+                                    }
+                                }
+                                else if (isflowControlParam == false)
+                                {
+                                    if (stepActivityVar is VariableSelectionList)
+                                    {
+                                        //change it to be string variable
+                                        stepActivity.Variables.Remove(stepActivityVar);
+                                        stepActivityVar = new VariableString();
+                                        stepActivityVar.Name = param;
+                                        ((VariableString)stepActivityVar).InitialStringValue = paramSelectedValue;
+                                        stepActivity.AddVariable(stepActivityVar);
+                                        stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because flow control param was removed
+                                    }
+                                }
+                            }
+
+                            //add the variable selected value                          
+                            if (stepActivityVar is VariableSelectionList)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary cast to 'Activity'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)